### ISSUE TITLE
build for scala 2.12 

### ISSUE THREAD

**Issue by jhpoelen** [ruippeixotog/scala-scraper#36]
Hi! Do you consider supporting scala 2.12 any time in the future?

See https://github.com/scala/make-release-notes/blob/2.12.x/projects-2.12.md

**Comment by ruippeixotog** [comment]
Hi! Yes, I'm on it.

Until yesterday I was waiting on http4s to release a version for Scala 2.12. Yesterday they released a snapshot (I only depend on http4s for testing, so I don't mind to use a snapshot temporarily) and all other dependencies already have stable versions for 2.12.

The problem now is that the 2.12.0 compiler crashes when compiling scala-scraper. I submitted [an issue](https://issues.scala-lang.org/browse/SI-10070), which I found out it's already fixed in the latest nightly version.

Bottom line, scala-scraper will have to wait until Scala 2.12.1 is released, which they intend to do at the end of November.

**Comment by jurisk** [comment]
http://www.scala-lang.org/news/2.12.1

**Comment by ruippeixotog** [comment]
Good to know! I'll get to release scala-scraper for 2.12 today if everything goes well.

**Comment by ruippeixotog** [comment]
I have just released scala-scraper 1.2.0, which was compiled for Scala 2.11 and 2.12.
